source/howtos/ssh_key_tutorial.rst

```diff
@@ -86,6 +86,9 @@ puttygen.exe generator.
   key. You will be asked for it when you connect via SSH.
 - Click "Save private key" to save your private key.
 - Click "Save public key" to save your public key.
-- Email your public key to support@osuosl.org
+- If you are creating this key for use with an OSL SSH account, copy and paste
```

Comment:
Can you call this out with a note block or however we do that sort of thing nowadays?  

I agree that this point needs emphasis.